mmv1/third_party/terraform/tests/resource_compute_firewall_policy_association_test.go

```diff
@@ -0,0 +1,58 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeFirewallPolicyAssociation_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+		"org_name":      fmt.Sprintf("organizations/%s", getTestOrgFromEnv(t)),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeFirewallPolicyAssociation_basic(context),
+			},
+			{
+				ResourceName:      "google_compute_firewall_policy_association.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccComputeFirewallPolicyAssociation_basic(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_folder" "folder" {
+  display_name = "tf-test-folder-%{random_suffix}"
+  parent       = "%{org_name}"
+}
+
+resource "google_folder" "target_folder" {
+  display_name = "tf-test-target-%{random_suffix}"
+  parent       = "%{org_name}"
+}
+
+resource "google_compute_firewall_policy" "default" {
+	parent      = google_folder.folder.name
+  short_name  = "tf-test-policy-%{random_suffix}"
+  description = "Resource created for Terraform acceptance testing"
+}
+
+resource "google_compute_firewall_policy_association" "default" {
+	firewall_policy = google_compute_firewall_policy.default.name
```

Comment:
Can we refer to the `id` field over `name` where possible, for consistency with the rest of the provider?